### DISCUSSION TITLE
How to open PDF in Adobe Acrobat Pro and Export to HTML

### DISCUSSION THREAD

**Discussion by dgraham-cra** [pywinauto/pywinauto#1268] [General]
I am trying to open a PDF file in Adobe Acrobat Pro (32-bit) and select "File->Export To->HTML Web Page" to export the PDF to HTML. See code snippets below.

For the life of me, I can't get the final two steps to work. I've tried various methods including menu_select ('File->Export To->HTML Web Page'), get_menu_path, and more. In addition, I've referred to various SO pages and this Reddit thread: https://www.reddit.com/r/learnpython/comments/92gaq3/pywinauto_adobe_reader_dc_how_do_i_invoke_a_menu/. I've also tried using Accessibility Insights, but I'm having difficulty translating what it shows (menu bar 'application'; menu item 'File'; menu 'File'; menu item 'Export To'; menu 'Export To'; menu item 'HTML Web Page') into a PyWinAuto script. 

I don't need to expand the various menus per se -- I just need to open a PDF at a given filepath and use Adobe's "Export To" functionality to export the PDF as HTML. Does Adobe not play nicely with PyWinAuto? This seems like a straightforward task, but it has proven to be quite frustrating.

If anyone has tips, I would be much obliged.

Thank you.

-------------------------------------------------------------

- [x] Open the PDF (using os.startfile for now)
```
os.startfile(PDF_file)`
```
- [x] Connect to the PDF
```
adobe = Application(backend="uia").connect(class_name='AcrobatSDIWindow')
window = adobe.window(title = PDF_filename)
```
- [x] Connect to the "Application" menu bar
```
app_menu = window.child_window(title="Application", control_type="MenuBar")
```
- [x] Expand "File" menu of "Application" menu bar
```
app_menu.child_window(title="File").expand()
```
- [] Expand "Extract To" submenu of "File" menu
```
???
```
- [] Click "HTML Web Page"
```
???
```

**Comment by vasily-v-ryabov** [comment]
Looks like expanded menu items appear in accessibility tree only after hovering mouse on any subitem. It's possible to use `.move_mouse_input()` with relatives coordinates up and down the main menu item to make sure the accessibility tree is updated. I have only Acrobat Reader, but I guess the behavior should be the same.
